docs/pages/teleport-editions.mdx

```diff
@@ -0,0 +1,115 @@
+|PCI DSS Features|Limited|✔|✔|
+|[Protecting Applications](./enroll-resources/application-access/getting-started.mdx)|✔|✔|✔|
+|[Protecting Databases](./enroll-resources/database-access/getting-started.mdx)|✔|✔|✔|
+|[Protecting Kubernetes Clusters](./enroll-resources/kubernetes-access/getting-started.mdx)|✔|✔|✔|
+|[Protecting Linux Servers](./enroll-resources/server-access/getting-started.mdx)|✔|✔|✔|
+|[Protecting Windows Desktops](./enroll-resources/desktop-access/introduction.mdx)|✔|✔|✔|
+|[Recording Proxy Mode](./enroll-resources/server-access/guides/recording-proxy-mode.mdx)|✔|✔|✖|
+|[Role-Based Access Control](./admin-guides/access-controls/guides/role-templates.mdx)|✔|✔|✔|
+|[Session Recording with Playback](./reference/architecture/session-recording.mdx)|✔|✔|✔|
+|[Single Sign-On](./admin-guides/access-controls/sso/sso.mdx)|GitHub|GitHub, Google Workspace, OIDC, SAML, Teleport|GitHub, Google Workspace, OIDC, SAML, Teleport|
+|SOC 2 Features|Limited|✔|✔|
+|[Structured Audit Logs](./reference/monitoring/audit.mdx)|✔|✔|✔|
+
+## Teleport Identity Governance
+
+Teleport Identity Governance hardens and monitors identities for both human and
+non-human identities. Teleport Identity Governance is bundled by default in
+Teleport Enterprise, and users can opt out when signing up for a Teleport
+Enterprise account.
+
+||Teleport Community Edition|Teleport Enterprise (Self-Hosted)|Teleport Enterprise (Cloud)|
+|---|---|---|---|
+|[Access Lists & Access Reviews](./admin-guides/access-controls/access-lists/access-lists.mdx)|✖|✔|✔|
+|[Access Monitoring & Response](./admin-guides/access-controls/access-monitoring.mdx)|✖|✔|✔|
+|[Device Trust](./admin-guides/access-controls/device-trust/guide.mdx)|✖|✔|✔|
+|[Endpoint Management: Jamf](./admin-guides/access-controls/device-trust/jamf-integration.mdx)|✖|✔|✔|
+|[Hardware Key Support](./admin-guides/access-controls/guides/hardware-key-support.mdx)|✖|✔|✔|
+|[Hardware Security Module support](./admin-guides/deploy-a-cluster/hsm.mdx) for encryption at rest|✖|✔|✖|
+|[JIT Access Requests](./admin-guides/access-controls/guides/dual-authz.mdx)|Limited|✔|✔|
+|[Session & Identity Locks](./admin-guides/access-controls/guides/locking.mdx)|✖|✔|✔|
+
+## Teleport Machine & Workload Identity
+
+Teleport Machine & Workload Identity is a solution for non-human identity
+management and access control, improving infrastructure resiliency by securing
+system and data access between machines and workloads. 
+
+Machine & Workload Identity is part of the Teleport Infrastructure Identity
+Platform. See [Usage and
+Billing](./usage-billing.mdx#machine-and-workload-identities) for how Teleport
+calculates usage of this product in addition to Teleport Enterprise usage.
+
+||Teleport Community Edition|Teleport Enterprise (Self-Hosted)|Teleport Enterprise (Cloud)|
+|---|---|---|---|
+|[Machine ID](./enroll-resources/machine-id/getting-started.mdx)|✖|✔|✔|
+|[Workload Identity](./enroll-resources/workload-identity/getting-started.mdx)|✖|✔|✔|
```

Comment:
Are we going to remove this stuff from community edition? Machine ID has been in CE since inception.

cc @strideynet